test/systemtests/network_test.go

```diff
@@ -81,6 +83,36 @@ func (s *systemtestSuite) testInfraNetworkAddDelete(c *C, encap string) {
 	}
 }
 
+/* Confirm network inspect includes a dns endpoint (assumes this is the only endpoint on the network currently) */
+func (s *systemtestSuite) testNetworkInspectDNS(tenant, network string) (bool, error) {
```

Comment:
I think i am sure :) The way test suite is designed is 
1) Testcase - basically consits of configs usage and calls util functions to help execute/verify/ the case. Test cases are supposed to be in respective test files.
2) Util functions: like the one you have added need to be in utils. This creates an abstraction on top of the kind of test environment you are running your test case. Your code will break k8s sanity (currently turned off due to a vagrant version issue). 
3) Actual test environment executors: Util functions will call these function depending on the test environment. If its k8s then calls k8s specific executors.If its docker then it calls docker specific executors. 

So you will need to move this to util all the way to executors. This will be a stub for k8s. I hope i have clarified the intention.